server/statistics/hot_peer_cache_test.go

```diff
@@ -121,34 +126,43 @@ func checkHit(c *C, cache *hotPeerCache, region *core.RegionInfo, kind FlowKind,
 	}
 }
 
+func checkNeedDelete(c *C, ret []*HotPeerStat, storeID uint64) {
+	for _, item := range ret {
+		if item.StoreID == storeID {
+			c.Assert(item.needDelete, IsTrue)
+			return
+		}
+	}
+}
+
 const interval = uint64(60)
 
-func schedule(operator operator, region *core.RegionInfo, kind FlowKind) *core.RegionInfo {
+func schedule(operator operator, region *core.RegionInfo, kind FlowKind) (srcStore uint64, _ *core.RegionInfo) {
```

Comment:
```suggestion
func schedule(operator operator, region *core.RegionInfo, kind FlowKind) (uint64, *core.RegionInfo) {
```

Reply:
I want to indicate the meaning of this return var, I think it is better than putting it in a comment

Reply:
> I want to indicate the meaning of this return var, I think it is better than putting it in a comment

I agree